integrations/terraform-cloud/.port/spec.yaml

```diff
@@ -0,0 +1,24 @@
+type: terraform-cloud
+description: terraform cloud integration for Port Ocean
+icon: Terraform # Should be one of the available icons in Port
+features:
+  - type: exporter
+    section: IaC # Should be one of the available sections in Port
+    resources:
+      - kind: workspace
+      - kind: run
+configurations:
+  - name: terraformHost
```

Comment:
Add a `default` value of https://app.terraform.io for the terraformHost

Reply:
added